CMakeLists.txt

```diff
@@ -152,7 +152,22 @@ endif()
 
 if(WIN32)
   message("-- Win32 build detected, setting default features")
-  set(USE_CAMERA_SUPPORT OFF)
+  set(USE_COLORD OFF)
+  set(USE_FLICKR OFF)
```

Comment:
Not available - yet, but I'm working on it. flickcurl has been compiled without any major problem, I need however to test it. The package is ready, just I don't want to add to the MINGW-packages until I test the flickcurl library properly. So first step; I need to create a Flickr account, as I'm not using it.